metecho/api/models.py

```diff
@@ -1067,6 +1067,21 @@ def finalize_provision(self, *, error=None, originating_user_id):
             else:
                 self.delete(originating_user_id=originating_user_id)
 
+    def queue_convert_to_dev_org(self, task, *, originating_user_id=None):
+        from .jobs import convert_to_dev_org_job
+
+        convert_to_dev_org_job.delay(
+            scratch_org=self, task=task, originating_user_id=originating_user_id
+        )
+
+    def finalize_convert_to_dev_org(self, task, *, originating_user_id):
+        self.org_type = SCRATCH_ORG_TYPES.Dev
+        self.task = task
+        self.epic = None
+        self.save()
+        task.notify_changed(originating_user_id=originating_user_id)
```

Comment:
@jerivas I don't think this line is needed, because the `_create_branches_on_github` job internally triggers a `task` update.